### PR TITLE
Demos for radio component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
#133

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos)

File: src/app/docs/radio/radio-docs.component.html [hunk 1]
```diff
@@ -7,8 +7,84 @@
...
+        </sky-column>
+        <sky-column screenSmall="6">
+          <h4>Icon radio group</h4>
+          <div
+            class="sky-switch-icon-group"
+          >
+            <sky-radio
+              icon="table"
+              label="Table"
+              name="iconRadioDemo"
```
Comment (blackbaud-johnly) [inline src/app/docs/radio/radio-docs.component.html:62]: Our docs say this property is deprecated in favore of the `name` property on `sky-radio-group`. Seems like we should either change this part of the demo to align with our recommendations in the docs or change the recommendation in the docs. Same question on the other `sky-radio` elements.

File: src/app/docs/radio/radio-docs.component.html [hunk 2]
```diff
@@ -7,8 +7,84 @@
     The radio button module ceates a small set of selection inputs for users to select one option.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
```
Comment (blackbaud-johnly) [inline src/app/docs/radio/radio-docs.component.html:10]: The current demo demonstrates both reactive and template-driven. Is demonstrating both no longer a priority?

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos)

File: src/app/docs/radio/radio-docs.component.html [hunk 3]
```diff
@@ -7,8 +7,84 @@
...
+    <sky-fluid-grid>
+      <sky-row>
+        <sky-column screenSmall="6">
+          <h4>Standard radio button</h4>
```
[the remark this answers is not in the document]
Comment (Blackbaud-AlexKingman) [reply src/app/docs/radio/radio-docs.component.html:14]: Done.

File: src/app/docs/radio/radio-docs.component.html [hunk 4]
```diff
@@ -7,8 +7,84 @@
...
+          <sky-radio-group
+            ariaLabel="Demo for radio button component"
```
[the remark this answers is not in the document]
Comment (Blackbaud-AlexKingman) [reply src/app/docs/radio/radio-docs.component.html:16]: Done.

File: src/app/docs/radio/radio-docs.component.html [hunk 5]
```diff
@@ -7,8 +7,84 @@
...
+          >
+            <ul
+              class="sky-list-unstyled"
+            >
+              <li>
+                <sky-radio
+                  value="selected"
+                >
+                  <sky-radio-label>
+                    A selected option
```
[the remark this answers is not in the document]
Comment (Blackbaud-AlexKingman) [reply src/app/docs/radio/radio-docs.component.html:28]: I just went with what was on the design spec, but you're right - its a little weird. I changed the options to match what was suggested for the code example: `Option 1` and `Option 2`.

File: src/app/docs/radio/radio-docs.component.html [hunk 6]
```diff
@@ -7,8 +7,84 @@
...
+                  </sky-radio-label>
+                </sky-radio>
+              </li>
+              <li>
+                <sky-radio
+                  value="disabled"
+                  [disabled]="true"
+                >
+                  <sky-radio-label>
+                    This option cannot be selected right now
```
[the remark this answers is not in the document]
Comment (Blackbaud-AlexKingman) [reply src/app/docs/radio/radio-docs.component.html:47]: I just matched the code example and went with `Option 3 is disabled`.

File: src/app/docs/radio/radio-docs.component.html (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/docs/radio/radio-docs.component.html:10]: The consumers don't have access to the code for the behavioral demo, so it doesn't really matter here. I just went with quick and easy.

File: src/app/docs/radio/radio-docs.component.html (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/docs/radio/radio-docs.component.html:62]: This code won't show to the consumer, but the code-example code will! I went to double check the code example, and removed the `name` property.

**Comment by Blackbaud-AlexKingman** [comment]
@blackbaud-johnly Steve approved, but I've seen to your comments. Could you please take another look?

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-radio-demos)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**